refarch-eai/src/main/resources/application-local.yml

```
@@ -2,4 +2,12 @@
 
 logging:
-  include-application-group: false
+  include-application-group: false # disabled because application group is only relevant when deployed
+  structured:
+    format:
+      console: # set to empty to disable structured logging locally
+  level:
+    root: INFO
+    de.muenchen.refarch: DEBUG
+    org.springframework.cache: TRACE
+    org.springframework.web: DEBUG
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)